DOL.WHD.Section14c.Api/Web.config

```diff
@@ -13,7 +13,7 @@
     <!--<add name="ApplicationDbContext" connectionString="Server=localhost:50000;Database=dolsec14;UID=db2inst1;PWD=!QAZ2wsx#EDC4rfv;CurrentSchema=DBO" providerName="IBM.Data.DB2" />-->
 
     <!-- Adding Devert DB2 -->
-    <add name="ApplicationDbContext" connectionString="Server=localhost:50000;Database=dolsec14;UID=db2admin;PWD=P@ssword1!" providerName="Devart.Data.DB2" />
+    <add name="ApplicationDbContext" connectionString="Server=localhost:50000;Database=dolsec14;UID=db2admin;PWD=!QAZ2wsx#EDC4rfv" providerName="Devart.Data.DB2" />
```

Comment:
It weirds me out a little bit every time I see the password in this file.  It's not a valid password to any active resources, right?  Just a dev password for a local dev environment?  If it's an active password, please cycle it ASAP.

Reply:
Server=localhost:50000.
That is my local server password.

Reply:
:thumbs-up:

Reply:
@mgwalker BTW, there is a defect to fix this issue slated to be in the next sprint.

Reply:
@mmurthydol Can you include the issue number here, just to keep everything cross-referenced?

Reply:
https://github.com/18F/dol-whd-14c/issues/186